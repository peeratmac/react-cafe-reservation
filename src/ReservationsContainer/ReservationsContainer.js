import React from 'react';
import ReservationsCard from '../ReservationsCard/ReservationsCard';

const ReservationsContainer = ({ reservations, fetchWithDelete }) => {
  return reservations.map(reservation => (
    <ReservationsCard
      key={reservation.id}
      guestName={reservation.name}
      date={reservation.date}
      time={reservation.time}
      numberOfGuests={reservation.number}
      id={reservation.id}
      fetchWithDelete={fetchWithDelete}
    />
  ));
};

export default ReservationsContainer;
