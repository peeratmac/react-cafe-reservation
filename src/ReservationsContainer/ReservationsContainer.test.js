import React from 'react';
import ReactDOM from 'react-dom';
import ReservationsContainer from './ReservationsContainer';
import { shallow } from 'enzyme';

describe('RESERVATIONS CONTAINER', () => {
  let wrapper, reservations;
  beforeEach(() => {
    reservations = [
      {
        name: 'Peerat',
        date: '12/25',
        time: '8:00',
        number: '555'
      }
    ];
    wrapper = shallow(<ReservationsContainer reservations={reservations} />);
  });

  it('should match ReservationsContainer Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
