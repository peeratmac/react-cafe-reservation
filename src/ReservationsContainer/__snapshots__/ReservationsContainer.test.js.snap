// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RESERVATIONS CONTAINER should match ReservationsContainer Snapshot 1`] = `
<ReservationsCard
  date="12/25"
  guestName="Peerat"
  numberOfGuests="555"
  time="8:00"
/>
`;
